submissions/phpslonuk/Popup/css/style.css

```diff
@@ -0,0 +1,222 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  max-width: 1170px;
+  margin: 0 auto;
+}
+
+header {
+  width: 100%;
+  height: 50px;
+  display: flex;
+  justify-content: space-between;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+/* Navigation */
+
+.nav {
+  position: fixed;
+  top: 10px;
+  right: 100px;
+}
+
+.nav_list {
+  list-style: none;
+  display: flex;
+}
+
+.nav_item {
+  margin: 10px;
+}
+
+.nav_item label{
+	cursor: pointer;
+}
+
+.nav_item:hover {
+  outline-style: none;
+  border-color: gray;
+}
+
+.nav_item_link {
+  padding-top: 5px;
+}
+
+.nav_item_mail, .nav_item_images {
+  font-size: 17px;
+  text-decoration: none;
+  color: #767676;
+}
+
+.nav_item_img {
+  width: 20px;
+  height: 20px;
+  margin-top: 3px;
+  border-radius: 10px;
+}
+
+#trigger {
+  appearance: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  position: absolute;
+}
+
+.popup_icon {
+  display: block;
+  background: url(https://raw.githubusercontent.com/phpslonuk/Hooli-style_Popup/main/popup-icons/popup-button.png) no-repeat;
+  background-position: left center;
+  background-size: auto 100%;
+  width: 26px;
+  height: 26px;
+}
+
+/* Check for correctness of work */
+
+input[type="checkbox"]:checked+.popup_icon {
+  border: 1px solid blue;
+}
+
+input[type="checkbox"]:checked~.popup_menu {
+  display: block;
+}
+
+/* Popup menu */
+
+.popup_menu {
+  display: none;
+  border: 1px solid black;
+  width: 310px;
+  height: 500px;
+  position: absolute;
+  top: 55px;
+  right: 10px;
+}
+
+.popup_menu::before {
+  content: '';
+  position: absolute;
+  width: 0;
+  height: 0;
+  border-left: 16px solid transparent;
+  border-right: 16px solid transparent;
+  border-bottom: 16px solid black;
+  z-index: -1;
+  top: -16px;
+  left: 198px;
+}
+
+.popup_menu::after {
+  content: '';
+  position: absolute;
+  width: 0;
+  height: 0;
+  border-left: 15px solid transparent;
+  border-right: 15px solid transparent;
+  border-bottom: 15px solid white;
+  top: -15px;
+  left: 199px;
+  z-index: 5;
+}
+
+.popup-content {
```

Comment:
would be better to use one style for all selector names (dash or underscore). Could you please fix it?